geoalchemy2/elements.py

```diff
@@ -219,6 +219,12 @@ def _data_from_desc(desc):
             desc = desc.encode(encoding="utf-8")
         return binascii.unhexlify(desc)
 
+    def __eq__(self, other):
+        return self.desc == other.desc
+
+    def __ne__(self, other):
+        return not self.desc == other.desc
```

Comment:
Maybe just call `__eq__` to avoid writing the test twice?
```suggestion
        return not self.__eq__(other)
```